controllers/sriovnetworkpoolconfig_controller.go

```diff
@@ -68,7 +76,7 @@ func (r *SriovNetworkPoolConfigReconciler) Reconcile(ctx context.Context, req ct
 				return reconcile.Result{}, err
 			}
 		}
-		if utils.ClusterType == utils.ClusterTypeOpenshift {
+		if utils.ClusterType == utils.ClusterTypeOpenshift && !isHypershift {
 			if err = r.syncOvsHardwareOffloadMachineConfigs(instance, false); err != nil {
```

Comment:
Your code skips `r.syncOvsHardwareOffloadMachineConfigs` when isHypershift, but there is nothing useful being logged when you try to enable hwol on hypershift. Can you please add logging?

Reply:
yep done!